fluentui_core/src/main/java/com/microsoft/fluentui/theme/token/controlTokens/AnnouncementCardTokens.kt

```diff
@@ -0,0 +1,116 @@
+package com.microsoft.fluentui.theme.token.controlTokens
+
+import android.os.Parcelable
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.microsoft.fluentui.theme.FluentTheme
+import com.microsoft.fluentui.theme.token.FluentAliasTokens
+import com.microsoft.fluentui.theme.token.FluentGlobalTokens
+import com.microsoft.fluentui.theme.token.StateColor
+import kotlinx.parcelize.Parcelize
+
+data class AnnouncementCardInfo(
+    override val cardType: CardType = CardType.Elevated,
+    val buttonStyle: ButtonStyle = ButtonStyle.TextButton
+) : BasicCardControlInfo()
+
+@Parcelize
+open class AnnouncementCardTokens : BasicCardTokens(), Parcelable {
+    @Composable
+    open fun textColor(announcementCardInfo: BasicCardControlInfo): Color {
```

Comment:
use the AnnouncementCardInfo

Reply:
using announcementCardInfo will not ovveride the base function.